packages/nativescript-exoplayer/index.android.ts

```diff
@@ -379,7 +379,7 @@ export class Video extends VideoBase {
 	pause() {
 		if (this.player) {
 			this.player.setPlayWhenReady(false);
-			this.player.stop();
+			// this.player.stop();
```

Comment:
I was thinking the same however I left it commented in case another report comes up around this for the next several weeks. I want to be sure there is no particular android version where the behavior may differ.